src/Adapter/CustomerService/CommandHandler/ReplyToCustomerThreadHandler.php

```diff
@@ -138,7 +138,7 @@ private function sendReplyEmail(CustomerThread $customerThread, CustomerMessage
         $contact = new Contact((int) $customerThread->id_contact, (int) $customerThread->id_lang);
 
         if (Validate::isLoadedObject($contact)) {
-            $fromName = $contact->name;
+            $fromName = is_array($contact->name) ? $contact->name[array_key_first($contact->name)] : $contact->name;
```

Comment:
```suggestion
            $fromName = $contact->name[array_key_first($contact->name)] ?? $contact->name;
```

Reply:
Array_key_first is only available since PHP 7.3 https://www.php.net/array_key_first
You can't use it.
Seems this part is not tested :thinking: